### PR TITLE
[Project] Replace old BuilderCfgFile by new BuilderCfgFile

### PR DESCRIPTION
**Pull request by YongseopKim**
let's replace old BuilderCfgFile by new BuilderCfgFile.

ONE-vscode-DCO-1.0-Signed-off-by: Yongseop Kim <<yons.kim@samsung.com>>

### CODE REVIEW

**Comment by YongseopKim** [comment]
After this pr, old BuilderCfgFile would be removed.

**Comment by YongseopKim** [comment]
But, first of all, https://github.com/Samsung/ONE-vscode/pull/528 should be necessary.

File: src/Project/BuilderCfgFile.new.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {EventEmitter} from 'events';
+import * as vscode from 'vscode';
+import {Command} from '../Backend/Command';
```
Comment (hyunsik-yoon) [inline src/Project/BuilderCfgFile.new.ts:19]: IMHO this class or job is not dependent on `Backend`. Probably how about putting `Command` into some common dir, for example, `common` (common to ONE-vscode and backend?) ?

**Review by jyoungyun — APPROVED** [review]
LGTM

**Pull request closed without merging**